jenkins/L0_MergeRequest.groovy

```diff
@@ -689,6 +695,43 @@ def getOnlyPytorchFileChanged(pipeline, testFilter, globalVars) {
     return result
 }
 
+def getOnlyTritonFileChanged(pipeline, testFilter, globalVars) {
+    def isOfficialPostMergeJob = (env.JOB_NAME ==~ /.*PostMerge.*/)
+    if (env.alternativeTRT || isOfficialPostMergeJob) {
+        pipeline.echo("Force set ONLY_TRITON_FILE_CHANGED false.")
+        return false
+    }
+    def tritonOnlyList = [
+        "tests/integration/defs/triton_server/",
+        "triton_backend/",
+    ]
+
+    def changedFileList = getMergeRequestChangedFileList(pipeline, globalVars)
+
+    if (!changedFileList || changedFileList.isEmpty()) {
+        return false
+    }
+
+    def result = true
+    for (file in changedFileList) {
+        def isTritonFile = false
+        for (prefix in tritonOnlyList) {
+            if (file.startsWith(prefix)) {
+                isTritonFile = true
+                break
+            }
+        }
+        if (!isTritonFile) {
+            pipeline.echo("Found non-Triton file: ${file}")
+            result = false
+            break
+        }
```

Comment:
If the first changed file is not in tritonOnlyList, then it will break here and return false